cpu/cc2538/periph/flashpage.c

```diff
@@ -61,16 +61,25 @@ static inline void _erase(uint32_t *page_addr)
     irq_restore(state);
 }
 
+void flashpage_erase(int page)
```

Comment:
Why not

```suggestion
void flashpage_erase(unsigned page)
```

Negative pages don't seem to be very useful.

Reply:
Just mimicking what is already used with the other functions. I don't mind changing them all to `unsigned`, including the `rwee` functions.

Reply:
btw I wonder if those RWWEE functions should live in `periph/flashpage.h` since they are sam0-specific.
For #14512 I just put the functions to write the AUX page into `sam0_common/include/periph_cpu_common.h`